CondCore/LuminosityPlugins/src/plugin.cc

```diff
@@ -1,9 +1,12 @@
 #include "CondCore/ESSources/interface/registration_macros.h"
 #include "CondFormats/Luminosity/interface/LumiSectionData.h"
 #include "CondFormats/DataRecord/interface/LumiSectionDataRcd.h"
+#include "CondFormats/DataRecord/interface/LumiCorrectionsRcd.h"
+#include "CondFormats/Luminosity/interface/LumiCorrections.h"
 
 
 REGISTER_PLUGIN(LumiSectionDataRcd, lumi::LumiSectionData);
```

Comment:
@capalmer85 why there is need of two records?

Reply:
I don't see two records.  This is the c++ definition:
CondFormats/Luminosity/interface/LumiCorrections.h

and this is the "record" object wrapper for it:
CondFormats/DataRecord/interface/LumiCorrectionsRcd.h

This is following the same structure as the two files listed before it.  I must be missing something.